errortypes/errortypes.go

```diff
@@ -12,6 +12,7 @@ const (
 	BidderTemporarilyDisabledCode
 	BlacklistedAcctCode
 	AcctRequiredCode
+	WarningCode
```

Comment:
This is just a question I have: Warning seems to be more generic than other code listed here. Should it rather be scoped to the currency warning? Something like `OneCurrencySupportedCode` or anything like that?

Reply:
The most generic error code is not having a code at all. The purpose of the error codes is to aid in automation, that is a client making the requests and receiving the responses can just look at the code rather than parse out the error message and behave accordingly. I would not have added the code at all except we have a piece of code that whitelists which error codes are not fatal codes, and the request can continue. So I felt a generic warning code would cover all the use cases and keep us from generating more codes in the future that will not be used for anything beyond adding to the non-fatal white list.

Reply:
Alright :)